addons/web/static/src/legacy/js/views/list/list_controller.js

```diff
@@ -445,6 +445,7 @@ var ListController = BasicController.extend({
             method: 'search',
             args: [domain],
             kwargs: {
+                context:this.model.get(this.handle, {raw: true}).getContext(),
```

Comment:
```diff
-                 context:this.model.get(this.handle, {raw: true}).getContext(),
+                 context: this.model.get(this.handle, { raw: true }).getContext(),
```

I guess another possibility would be to pass the context as a parameter but this seems OK too.

Reply:
Formatting fixed